qutebrowser/browser/webengine/webview.py

```diff
@@ -238,3 +238,12 @@ def acceptNavigationRequest(self,
             is_main_frame=is_main_frame)
         self.navigation_request.emit(navigation)
         return navigation.accepted
+
+    def chooseFiles(self, mode, old_files, accepted_mimetypes):
```

Comment:
Mind adding type annotations here as well? Something like:

```suggestion
    def chooseFiles(self, mode: QWebEnginePage.FileSelectionMode, old_files: List[str], accepted_mimetypes: List[str]) -> List[str]:
```

(untested)